src/Microsoft.Health.Dicom.Api/Features/BackgroundServices/StartInstanceBlobMigrationService.cs

```diff
@@ -0,0 +1,65 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Features.Operations;
+
+namespace Microsoft.Health.Dicom.Api.Features.BackgroundServices;
+
+public class StartInstanceBlobMigrationService : BackgroundService
```

Comment:
As this service only start the copy orchestration, how about `StartCopyService`?
Or maybe you have different plan?

Reply:
For now keeping the same service for both. We can change if we have two service